docs/make_aggregate.jl

```diff
@@ -1,7 +1,9 @@
 using Documenter, LibGit2, Pkg
 using MultiDocumenter
 
-clonedir = mktempdir()
+include("CommercialSupportComponent.jl")
+
+clonedir = joinpath(@__DIR__, "cloned")
```

Comment:
This makes repeated builds on a local machine significantly faster, and avoids cluttering the temp directory with 14 GB of files :D